### PR TITLE
Add aliases to the security_token param

### PR DESCRIPTION
**Pull request by zollo**
##### SUMMARY
This pull request adds two aliases to the security_token param to make it more in-line with the correct AWS terminology ("session token") - the Boto SDK even refers to it this way.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
AWS Common Argument Spec / _aws_common_argument_spec()

### CODE REVIEW

**Comment by markuman** [comment]
Thanks for that. Can you also need a changelog fragment?

**Comment by markuman** [comment]
recheck

**Comment by jillr** [comment]
This lgtm when it has a changelog

**Comment by jillr** [comment]
recheck

**Comment by alinabuzachis** [comment]
@zollo Thank you for taking time to work on this. I agree with @jillr for the changelog. In addition, If I am not mistaken the aliases needs to be documented here https://github.com/ansible-collections/amazon.aws/blob/main/plugins/doc_fragments/aws.py#L52 (just extend the aliases section with the new ones).

**Comment by zollo** [comment]
> @zollo Thank you for taking time to work on this. I agree with @jillr for the changelog. In addition, If I am not mistaken the aliases needs to be documented here https://github.com/ansible-collections/amazon.aws/blob/main/plugins/doc_fragments/aws.py#L52 (just extend the aliases section with the new ones).

Thank you so much - i've corrected this!

**Review by alinabuzachis — APPROVED** [review]
@zollo Please also add a changelog fragment as Jill suggested https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs. Other than that, this LGTM! Thank you.

**Comment by zollo** [comment]
> @zollo Please also add a changelog fragment as Jill suggested https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs. Other than that, this LGTM! Thank you.

Done! Thank you!

**Review by alinabuzachis — APPROVED** [review]
@zollo thank you!

File: plugins/doc_fragments/aws.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ class ModuleDocFragment(object):
       - Passing the I(security_token) and I(profile) options at the same time has been deprecated
         and the options will be made mutually exclusive after 2022-06-01.
     type: str
-    aliases: [ aws_security_token, access_token ]
+    aliases: [ aws_session_token, session_token, aws_security_token, access_token ]
```
Comment (Akasurde) [inline plugins/doc_fragments/aws.py:52]: I would add a note in the description section stating that these new aliases are added in a particular version. For example like this https://github.com/ansible-collections/community.vmware/blob/c571d0b97abb9ba9032a3bb6879624f00365650b/plugins/modules/vmware_host_vmnic_info.py#L88

**Comment by alinabuzachis** [comment]
recheck

**Pull request merged**